content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu. 
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading. 
+
+## Licenses
+
+The Licenses tab features a Consumption trend chart of data for the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how may licenses were available that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline of shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection. 
```

Comment:
Typo in "may" - also, let's update this phrase to "see how many licenses were being utilized that month"